src/components/structural/Textures.js

```diff
@@ -1,10 +1,11 @@
 let TexturePack = [
     {title:"bricks",url:"/img/textures/bricks.jpg"},
     {title:"bark",url:"/img/textures/bark.jpg"},
-    {title:"checkerboard",url:"/img/textures/checkerboard.png"},
+    {title:"checkerboard",url:"/img/textures/checkerboard.jpg"},
     {title:"chevron",url:"/img/textures/chevron.jpg"},
     {title:"cobblestone",url:"/img/textures/cobblestone.jpg"},
     {title:"dirt",url:"/img/textures/dirt.jpg"},
+    {title:"duck",url:"/img/textures/duck.jpg"},

```

Comment:
It seems like you're changing 2 images here, I don't see a backend PR to change those.